src/test/suite/participant/participant.test.ts

```diff
@@ -14,8 +14,11 @@ import ConnectionController from '../../../connectionController';
 import { StorageController } from '../../../storage';
```

Comment:
Reviewing this file with whitespace changes ignored makes it easier to see the actual changes.